railties/test/generators/app_generator_test.rb

```diff
@@ -244,12 +244,12 @@ def test_app_update_does_not_generate_assets_initializer_when_sprockets_and_prop
     assert_no_file "app/assets/config/manifest.js"
   end
 
-  def test_app_update_does_not_generate_manifest_config_when_propshaft_is_used
-    run_generator [destination_root, "-a", "propshaft"]
+  def test_app_update_does_not_generate_manifest_config_when_sprockets_is_used
```

Comment:
Isn't this test time wrong now? I think this test is making sure the `manifest.js` file isn't deleted right?

Reply:
Removing since this test already exists:
https://github.com/zzak/rails/blob/12a625e5cb682b2c180740be6239dec33291b4f1/railties/test/generators/app_generator_test.rb#L331-L340